### PR TITLE
Add 2930 tests

### PR DESCRIPTION
**Pull request by sunny2022da**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,15 @@ The genesis config should define the block heights for `berlinBlock`, `londonBlo
     }
 ```
 
+To test EIP-2930, the following memory address and contents are set in the `alloc` section of the genesis config to simplify the test.
+```
+      "0x000000000000000000000000000000000000aaaa" : {
```
Comment (Mister-EA) [inline README.md]: Please add it to the genesis file also

File: eip2930/tests.go [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+	testCases := []TestCase{
+		{
+			name:               "testSendAccessListTx",
+			validationFunction: testSendAccessListTx,
+		},
+	}
+	runTestCasesSequentially(testCases)
+}
+
+func postHertzTests() {
```
Comment (Mister-EA) [inline eip2930/tests.go:219]: There should be preHertzTests() also, where the `AccessListTx` fails.

**Review by Mister-EA — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sunny2022da) [reply README.md]: Thanks to point it out. This is incorrect actually, the address here should be `0x7B31188CA9C1374AC9174C3D1F23F98180CBB67C` which is added to the genesis. I will fix it.

File: eip2930/tests.go [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+	log.Println("Pre-Hertz tests:")
+	blockNr, err := client.BlockNumber(context.Background())
+	if err != nil {
+		log.Fatal(err)
+	}
+	if blockNr >= config.PostHertzBlockNumber {
+		log.Fatalf("Too late to run pre-Hertz tests since current block number %v is after Hertz hard fork block %v.\n", blockNr, config.PostHertzBlockNumber)
+	}
+	log.Printf("Waiting for block number %v to start running the test cases...\n", config.PreHertzBlockNumber)
+	runPreHertz2930Tests()
```
Comment (Mister-EA) [inline eip2930/tests.go:193]: The errors from `runPreHertz2930Tests()` are ignored.

**Review by Mister-EA — CHANGES_REQUESTED** [review]

File: eip2930/tests.go (hunk 3 above)
Comment (Mister-EA) [inline eip2930/tests.go]: The errors from `runPreHertz2930Tests()` are ignored.

**Review by Mister-EA — CHANGES_REQUESTED** [review]

**Pull request merged**